cpp/src/ray/runtime/task/task_executor.cc

```diff
@@ -135,7 +135,8 @@ Status TaskExecutor::ExecuteTask(
     bool *is_application_error,
     const std::vector<ConcurrencyGroup> &defined_concurrency_groups,
     const std::string name_of_concurrency_group_to_execute,
-    bool is_reattempt) {
+    bool is_reattempt,
+    std::string *task_execution_error) {
```

Comment:
is this always the application error (exceptions), or does it include other type of errors (e.g., sys.exit)?

Reply:
It will be applicatoin error + system error: 

https://github.com/ray-project/ray/blob/477e6d4ca4a13c196657e1dd1187a954fa12e03f/src/ray/core_worker/transport/direct_actor_transport.cc#L140-L149

Reply:
oh I meant the value returned from raylet.pyx. Is this application error only? If so, it may be clearer to just call it application_error_message or sth?

Reply:
oh i see. yes, good point.

Reply:
are you going to address this comment? Haven't seen it changed yet